io.github.shiftey.Desktop.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.shiftey.Desktop</id>
+  <name>Github-desktop</name>
+  <summary>GitHub Desktop is an open source GitHub app</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <description>
+    <p>
+      GitHub Desktop is an open source GitHub app. This version of Github Desktop is a fork that adds support for Linux
+    </p>
+  </description>
+  <url type="homepage">https://github.com/shiftkey/desktop#readme</url>
+  <launchable type="desktop-id">io.github.shiftey.Desktop.desktop</launchable>
+  <screenshots>
+        <screenshot type="default">
+          <image>https://cloud.githubusercontent.com/assets/359239/26094502/a1f56d02-3a5d-11e7-8799-23c7ba5e5106.png</image>
+        </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1" />
+  <releases>
+        <release version="2.8.0" date="2020-7-14">
```

Comment:
There's actually a 2.9.0 release that's more current than this: https://github.com/shiftkey/desktop/releases/tag/release-2.9.0-linux1

Reply:
my bad I haven't updated the metainfo file